README.md

```diff
@@ -118,45 +99,28 @@ Feel free to use the following as resources:
 - [Machine Learning Fundamentals: Cross Validation](https://www.youtube.com/watch?v=fSytzGwwBVw&pp=ygUYdmFsaWRhdGlvbiBzZXQgc3RhdHF1ZXN0)
 - [Bootstrapping Main Ideas!!!](https://www.youtube.com/watch?v=Xz0x-8-cgaQ&pp=ygUXYm9vdHN0cmFwcGluZyBzdGF0cXVlc3Q%3D)
 - [Bootstrapping Method](https://www.youtube.com/watch?v=uGsf3spCM3Y)
-- [$k$-fold Cross Validation](https://www.youtube.com/watch?v=TIgfjmp-4BA)
-
-Alternative Textbook: [Data Science: A First Introduction](https://python.datasciencebook.ca/classification1.html) (Chapters 5-10)
+- [k-fold Cross Validation](https://www.youtube.com/watch?v=TIgfjmp-4BA)
 
 ### How to get help
 ![image](./steps_to_ask_for_help.png)
+Please remember to use the help channel to get help from the course support!
 
 <hr>
 
 ## Folder Structure
 
 ```markdown
 .
-├── .github
-├── .gitignore
 ├── 01_materials
 ├── 02_activities
+│   │   assignments
+│   │   exercises
+│   │   homework
 ├── 03_instructional_team
-├── 04_cohort_three
+├── 04_cohort_4
 ├── LICENSE
 ├── README.md
 └── steps_to_ask_for_help.png
 ```
 
-* **slides:** Module slides as PDF files.
-* **exercises:** Work to be done alongside learning sessions.
-* **homework:** Homework to practice concepts covered in learning modules.
-* **assignments:** Assignments.
-* **additional resources:** Extra material not covered by the module.
-* **instructors:** This folder provides guidance for Technical Facilitators and the Learning Support team on teaching methodologies and content delivery.
-* README: This file!
-* .gitignore: Files to exclude from this folder, specified by the Technical Facilitator
-
-* **.github**: Contains issue templates and pull request templates for the repository.
-* **materials**: Module slides (.pdf) used during learning sessions.
-* **activities**: Contains graded assignments, exercises, and homework to practice concepts covered in the learning module.
-* **instructional_team**: Resources for the instructional team.
-* **cohort_three**: Additional materials and resources for cohort three.
-* **.gitignore**: Files to exclude from this folder, specified by the Technical Facilitator
-* **LICENSE**: The license for this repository.
-* **README.md**: This file.
-* **steps_to_ask_for_help.png**: Guide on how to ask for help.
\ No newline at end of file
+Exercises and homework are **optional** additional resources for you to learn outside the live learning sessions. Only assignments are mandatory in order to successfully pass this module.
```

Comment:
There are no more "exercise" or "homework", just "slides" and "notebook", but yes the assignment part is correct.